jobs/cc_deployment_updater/spec

```diff
@@ -51,6 +51,9 @@ consumes:
   optional: true
 - name: cloud_controller_internal
   type: cloud_controller_internal
+- name: active_signing_key
```

Comment:
https://github.com/cloudfoundry-incubator/bits-service-release/blob/af3ee13beab8c6b2942c9e5bab7c7e7b5b4272f4/jobs/bits-service/spec#L27

Is it too late for this link to be renamed? It's unclear what it actually belongs to, so it would be nice if the link was prefixed with `bits_service` or something.

Reply:
Hey @tcdowney, No it's not too late. Will make the change.

Reply:
Great, thank you!